package/rules/swc.js

```diff
@@ -0,0 +1,73 @@
+const { resolve } = require('path')
+const { realpathSync } = require('fs')
+
+const {
+  source_path: sourcePath,
+  additional_paths: additionalPaths
+} = require('../config')
+const { isProduction } = require('../env')
+
+module.exports = [
+  {
+    test: /\.(js|jsx|mjs|coffee)?(\.erb)?$/,
+    include: [sourcePath, ...additionalPaths].map((p) => {
+      try {
+        return realpathSync(p)
+      } catch (e) {
+        return resolve(p)
+      }
+    }),
+    exclude: /node_modules/,
+    use: [
+      {
+        loader: require.resolve('swc-loader'),
+        options: {
```

Comment:
Options here correspond to https://swc.rs/docs/configuration/compilation Can put them in `.swcrc` file also (will support different flows based on file extension)